### ISSUE TITLE
Duplicate fields in Pinger struct

### ISSUE THREAD

**Issue by darwish** [sparrc/go-ping#25]
This merge request appears to have created some duplicate fields: https://github.com/sparrc/go-ping/pull/15

```bash
duplicate field name in struct literal: network
duplicate field name in struct literal: ipv4
```

**Comment by jwriteclub** [comment]
And this is why we need `vgo`.

@sparrc please merge as soon as you're able.

**Comment by inhuman** [comment]
@sparrc we wait

**Comment by sparrc** [comment]
sorry about that, fixed by #27
